src/main/java/com/team1678/frc2024/Robot.java

```diff
@@ -87,15 +88,14 @@ public class Robot extends TimedRobot {
 
 	double disable_enter_time = 0.0;
 
-	static {
-		if (Robot.isReal()) {
-			serial = System.getenv("serialnum");
-		} else {
-			serial = "";
-		}
-		Constants.isComp = serial.startsWith(Constants.kCompSerial);
-		Constants.isEpsilon = serial.startsWith(Constants.kEpsilonSerial);
-	}
+	// static {
+	// 	if (Robot.isReal()) {
+	// 		serial = System.getenv("serialnum");
+	// 	} else {
+	// 		serial = "";
+	// 	}
+	// 	Constants.isComp = serial.startsWith(Constants.kCompSerial);
+	// }
```

Comment:
Remove commented code. We don't ever plan to implement this environment variable system they have. If we want something similar, I have a better solution that the Bees use, but we don't need it right now for BoB.